frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeaderInfo.tsx

```diff
@@ -20,7 +20,7 @@ const StyledHeaderMetaInfo = styled('div')(({ theme }) => ({
     },
 }));
 
-const StyledExpandItem = styled('div')(({ theme }) => ({
+const StyledExpandItem = styled('p')(({ theme }) => ({
```

Comment:
Text is usually better with paragraphs. This one is kinda optional. I was unsure about whether we want to hide this for screen readers or not, but I think it's fine as is.